Kitodo/src/main/java/de/sub/goobi/forms/ProjekteForm.java

```diff
@@ -800,4 +843,124 @@ public int getItemId() {
         return this.itemId;
     }
 
+    /**
+     * Get the XML configuration string.
+     *
+     * @return the XML configuration string
+     */
+    public String getXMLConfiguration() {
+        return this.xmlConfigurationString;
+    }
+
+    /**
+     * Set the XML configuration string.
+     *
+     * @param configuration
+     *            the XML configuration string
+     */
+    public void setXMLConfiguration(String configuration) {
+        this.xmlConfigurationString = configuration;
+    }
+
+    /**
+     * Load the content of the XML configuration file denoted by given String
+     * 'configurationName'. This must be one of the keys in the static HashMap
+     * 'configurationFiles'.
+     *
+     * @param configurationName
+     *            name of the configuration to be loaded
+     */
+    public void loadXMLConfiguration(String configurationName) {
+        try (StringWriter stringWriter = new StringWriter()) {
+            currentConfigurationFile = configurationFiles.get(configurationName);
+            XMLConfiguration currentConfiguration = new XMLConfiguration(
+                    ConfigCore.getKitodoConfigDirectory() + currentConfigurationFile);
+            currentConfiguration.save(stringWriter);
+            this.xmlConfigurationString = stringWriter.toString();
+        } catch (ConfigurationException e) {
+            String errorMessage = "ERROR: Unable to load configuration file for '" + configurationName + "'.";
+            logger.error(errorMessage + " " + e.getMessage());
+            this.xmlConfigurationString = errorMessage;
+        } catch (IOException e) {
+            logger.error(e.getMessage());
+        }
+    }
+
+    /**
+     * Save the String 'xmlConfigurationString' to the XML file denoted by
+     * 'configurationFile'.
+     */
+    public void saveXMLConfiguration() {
+        try {
+            Document document = documentBuilder.parse(new InputSource(new StringReader(this.xmlConfigurationString)));
+            TransformerFactory transformerFactory = TransformerFactory.newInstance();
+            Transformer transformer = transformerFactory.newTransformer();
+            DOMSource domSource = new DOMSource(document);
+            File xmlConfigurationFile = new File(ConfigCore.getKitodoConfigDirectory() + currentConfigurationFile);
+            FileOutputStream outputStream = new FileOutputStream(xmlConfigurationFile, false);
```

Comment:
For `xmlConfigurationFile` and `outputStream`objects you should even use the try-with-resources statement.